static/scss/share-popup.scss

```diff
@@ -46,3 +36,66 @@
     }
   }
 }
+
+.share-popup {
+
+  .triangle {
+    width: 15px;
+    height: 15px;
+    position: absolute;
+    top: -7px;
+    background: white;
+    transform: rotate(45deg);
+    right: 50%;
+  }
+}
+
+.login-popup {
+  top: unset;
+  right: unset;
+  margin-top: 10px;
+
+  .triangle {
+    width: 15px;
+    height: 15px;
+    position: absolute;
+    top: -7px;
+    background: white;
+    transform: rotate(45deg);
+  }
```

Comment:
some of this styling for `.triangle` is the same as for `.share-popup` above, can we consolidate the properties that are the same?